Formula/ale.rb

```diff
@@ -0,0 +1,41 @@
+class Ale < Formula
+  desc "The Arcade Learning Environment (ALE) -- a platform for AI research."
+  homepage "https://github.com/mgbellemare/Arcade-Learning-Environment"
+  url "https://github.com/mgbellemare/Arcade-Learning-Environment/archive/v0.5.2.tar.gz"
+  sha256 "bc158b4fb8edc5034f8f8b4ca1a90d48590f516b33f9edec325fce8762418069"
+
+  head do
+    url "https://github.com/mgbellemare/Arcade-Learning-Environment.git"
+  end
```

Comment:
You can reduce this to `head "https://….git"`

Reply:
Will fix

Reply:
Will fix

Reply:
Will fix